RabbitMq/Consumer.php

```diff
@@ -256,13 +278,17 @@ public function getGracefulMaxExecutionTimeoutExitCode()
     }
 
     /**
-     * Choose the timeout to use for the $this->getChannel()->wait() method.
+     * @return int
+     */
+    public function getTimeoutWait()
```

Comment:
Please return type here and not in comment, as: 
public function getTimeoutWait(): int